Misc/NEWS.d/next/Library/2024-08-19-09-36-50.gh-issue-123135.w_ZNAs.rst

```diff
@@ -0,0 +1,3 @@
+Added support for supplying :term:`path-like objects <path-like object>`
+to the *names* parameter of :func:`fnmatch.filter`. Previously, such
+objects were only accepted on non-POSIX platforms. Patch by Bénédikt Tran.
```

Comment:
The only non-POSIX platform that we support is Windows:

```suggestion
objects were only accepted on Windows. Patch by Bénédikt Tran.
```

Reply:
What about WASI?